templates/webpack-default/app/README

```diff
@@ -0,0 +1,31 @@
+Welcome to Alloy. Prepare to be amazed.
```

Comment:
The default webpack template contains a `README.md` so having an additional `README` is probably confusing. I aligned the `README.md` for the new Angular and webpack classic templates so they are bascially the same, just point to different guides where to get help. Would be nice if the Alloy readme looks the same.

Reply:
Updated the `alloy new` command to append to/create the `README.md` file if one exists in the app template - 7d82a5c67c7ecbf6fe0b0dd3c6ee264257356e09